internal/servers/repository_worker_test.go

```diff
@@ -400,11 +444,13 @@ func TestUpsertWorkerStatus(t *testing.T) {
 			_, err = tc.repo.UpsertWorkerStatus(ctx, tc.status)
 			assert.Error(t, err)
 			tc.errAssert(t, err)
+			t.Log("error: ", err)
```

Comment:
Fine if you keep these two t.Log statements in, just want to check that it's on purpose and not on accident.

Reply:
yes, intentional.